frontend/packages/dev-console/src/components/import/devfile/devfileHooks.ts

```diff
@@ -56,6 +57,17 @@ export const useDefileServer = (
         if (value) {
           clearError();
           const { imageStream, buildResource, deployResource, service, route } = value;
+
+          if (deployResource.spec?.template?.spec?.containers?.length > 0) {
+            const buildGuidanceContainer = deployResource.spec.template.spec.containers[0];
+
+            setFieldValue('image.ports', buildGuidanceContainer.ports || [], false);
+            setFieldValue('deployment.env', buildGuidanceContainer.env || [], false);
```

Comment:
Today it does not matter (they are the same value)... but I am wondering if we should refactor `getBaseInitialValues` to allow for you to use the static settings for `ports` and `env` and then use those here instead of empty. 

Since it's unlikely to change, I think we can avoid holding this PR up for that change... food for thought though since what you're really doing here is a reset to defaults.